src/Formula/Parsing.hs

```diff
@@ -82,6 +83,15 @@ instance Parse a => Parse [a] where
         tokenSymbol "]" <|> fail "could not parse an enclosing ']'"
         pure xs
 
+instance (Parse a, Parse b) => Parse (a, b) where
+  parser = do
+    tokenSymbol "("
+    l <- parser
+    tokenSymbol ","
+    r <- parser
+    tokenSymbol ")"
+    pure (l, r)
```

Comment:
Hier wäre wahrscheinlich "syntaktische Fehlertoleranz" bzw. "Erklärungen zu typischen syntaktischen Fehlern" wie oben in der `Parse [a]`-Instanz sinnvoll.

Reply:
I removed the new parse instance since we are using lists now.